CHANGELOG.md

```diff
@@ -32,7 +32,11 @@
 
     Currently, you can only configure things related to the above-mentioned build profiles. To learn more, 
     [check out the documentation][profile-config-docs]. It leverages the `package.metadata.wasm-pack` key in your
+<<<<<<< HEAD
```

Comment:
rebasing seems not finished yet.

Reply:
changelog is checked and fixed. Any issue else?

Reply:
no more :)